docker/docker_client.go

```diff
@@ -78,7 +78,7 @@ func newDockerClient(refHostname, certPath string, tlsVerify bool) (*dockerClien
 	}, nil
 }
 
-func (c *dockerClient) makeRequest(method, url string, headers map[string]string, stream io.Reader) (*http.Response, error) {
+func (c *dockerClient) makeRequest(method, url string, headers map[string][]string, stream io.Reader) (*http.Response, error) {
```

Comment:
I firstly used http.Header but later changed just because I wanted to keep this simple (wrt types) because we might change for instance the `http` pkg (hard we do this lol) and don't want to change that all over the place - so a map[string][]string is generic enough :smile: